app/assets/javascripts/application/slides.js

```diff
@@ -1,12 +1,17 @@
 Utils.fireWhenReady(['slides#preview'], function(e) {
   // Scale the view to the size it will need to display
-  $('.ui-feed').css({width: '1920px', height: '1080px'});
+  var $uiFeed = $('.ui-feed');
+  var isVertical = $uiFeed.find(".js-slide-meta").data("orientation") == "vertical" ? true : false;
+  var stdWidth = isVertical? 1080 : 1920;
+  var stdHeight = isVertical? 1920 : 1080;
+
+  $('.ui-feed').css({width: stdWidth, height: stdHeight});
```

Comment:
Does it matter that the `width` and `height` values passed here used to be strings with `px` attached to end and are now integers?

Nope. Looks like jQuery accepts both:

- http://stackoverflow.com/questions/2722068/how-to-set-width-and-height-dynamically-using-jquery